packages/mobile/src/analytics/ValoraAnalytics.ts

```diff
@@ -72,7 +72,7 @@ class ValoraAnalytics {
         this.deviceInfo = deviceInfo
         this.sessionId = Web3Utils.soliditySha3({
           type: 'string',
-          value: deviceInfo.SerialNumber || deviceInfo.DeviceId,
+          value: Math.random().toString(),
```

Comment:
Hadn't noticed this before but could DeviceInfo.getUniqueId() work as this identifier?

Reply:
Don't we want sessionId to change with each app load? My understanding is that sha256 will output a constant hash when given a constant input.